onebusaway-android/src/androidTest/res/raw/nav_trip29.csv

```diff
@@ -633,4 +633,3 @@ Hillsborough Area Regional Transit_342999,Hillsborough Area Regional Transit_743
 631,true,true,412193795086171,1534348755000,28.069240,-82.412334,-23.595184,13.430000,271.299988,12.136000,15,gps
 632,true,true,412193795086171,1534348755000,28.069240,-82.412334,-23.595184,13.430000,271.299988,12.136000,15,gps
 633,true,true,412193795086171,1534348755000,28.069240,-82.412334,-23.595184,13.430000,271.299988,12.136000,15,gps
-
```

Comment:
@paulnabanita1 There was an extra newline at the end of one of the test files (nav_trip29.csv) that was causing that test to fail when trying to parse the empty line.  I added another commit to this PR to test and verify that this was the problem, so you don't need to do anything for this PR.  Just FYI, so you know to check future test files and make sure they don't have extra space at the end.  We should also update `NavigationTest` so it doesn't fail on these types of files - I added another checkbox for this to https://github.com/OneBusAway/onebusaway-android/pull/873, and we can tackle it later.